packages/waku/src/router/client.ts

```diff
@@ -808,7 +808,8 @@ export function NewRouter({
               // FIXME this check here seems ad-hoc (less readable code)
               if (
                 window.location.pathname !== path ||
-                window.location.search.replace(/^\?/, '') !== query
+                (!isStatic &&
+                  window.location.search.replace(/^\?/, '') !== query)
```

Comment:
are we sure the `.replace` is still needed?

Reply:
nvm I just doubled checked... it is needed

Reply:
I think `window.location.search` has `?`, while `query` does not, no?

Reply:
you're right. I was mistaken with my initial comment, please disregard.